### DISCUSSION TITLE
What categories should we have here?

### DISCUSSION THREAD

**Discussion by BenTheElder** [kubernetes/sig-testing#3] [General]
The default categories are:

- 💬 • General • Chat about anything and everything here

- 💡 • Ideas • Share ideas for new features

- 🙏 • Q&A • _Answers enabled ✅_   ·  Ask the community for help
  - _Answers Enabled ✅_  means comments are sorted by highest upvoted by default and we can mark accepted answers, similar to StackOverflow for this discussion type. Other discussion types are sorted by comment age by default but users can change this, and do not have "mark answer". Any custom issue type can have this mode enabled.
   
![image](https://user-images.githubusercontent.com/917931/108911647-630d4100-75dc-11eb-9900-6af91c7214e9.png)

- 🙌 • Show and tell • Show off something you've made

I think it makes sense to retain at:
- 🙏 • Q&A • For what it says. This could be nice versus slack to have linkable question & answer with accepted answers without needing to get people on stackoverflow or treat these like normal issues.

- 💬 • General • For things that don't fall neatly into other categories

But I'm not sure how useful "Ideas" / "Show and Tell" are versus other categories we might come up with.

It also may make sense to divide up Q&A to specific categories or ask some of our projects do enable discussions to have project specific Q&A where applicable.

What categories do you think we should have?

**Comment by parispittman** [comment]
I think Q&A and General will get you to where you need to go and then grow from there.

**Reply by parispittman** [reply]
ooo just thought of a word - what about brainstorm? I've been thinking about the conflicted thoughts of `idea` and think that brainstorm might be more of where I'm going.

**Reply by BenTheElder** [reply]
cc @spiffxp WDYT?

**Reply by spiffxp** [reply]
IMO idea==brainstorm but maybe that's just me

- idea: proposal / brainstorm / what if we...
- general: iterating on an idea we've agreed is worth doing, misc.
- q&a: support with linkable answers

until volume is crazy high I'm fine routing / modifying categories appropriately

**Reply by spiffxp** [reply]
oh wait I see, you're both saying start with general/q&a.... yeah, fewer better!  wait to add more complexity when we've found we need it

**Comment by spiffxp** [comment]
So I did a thing.  Good enough to close this out for now?
![Screen Shot 2021-03-19 at 1 41 00 PM](https://user-images.githubusercontent.com/49258/111821182-bfe8d800-88b8-11eb-92d0-753065b6f444.png)

... right and then re: "closing out", should I convert to Q&A and consider this "the answer"? just leave this open? no strong opinions
